.pipelines/cni/cilium/cilium-overlay-load-test-template.yaml

```diff
@@ -235,6 +269,7 @@ stages:
               scaleup: ${SCALEUP_CILIUM}
               nodeCount: ${{ parameters.nodeCount }}
               cni: ${{ parameters.cni }}
+              upgradeScenario: ${{ parameters.upgradeScenario }}

```

Comment:
should we also pass the logType parameter here instead of upgradeScenario, similar to how you did it above?